test/comparison/allocation/allocation_comparison_test.go

```diff
@@ -130,11 +133,15 @@ func compareAllocationResponses(t *testing.T, resp1, resp2 *api.AllocationRespon
 func TestAllocationAPIComparison(t *testing.T) {
 	// Initialize both API clients
 	api1 := api.NewAPI()
-	api2 := api.NewAPI()
+	api2 := api.NewComparisonAPI()
+
+	end := time.Now().UTC().Truncate(10 * time.Minute)
```

Comment:
don't you only want to truncate this to 1 minute? Let's say that our stack was deployed at 10:14 AM and promless started collecting data immediately. We wait 15 minutes, it is now 10:29 AM. Let's say this test then runs immediately. My reading of the truncate is then that your window is going to be 10:10 AM start to 10:20 AM end? won't promless then only have data starting 4 mins into the window?

Reply:
I'm open to increasing the wait time to 20 mins. can I PR into this branch?

Reply:
yeah definitely